### PR TITLE
Fix incorrect font base size 

### LINKED ISSUE #30367: [Issue] Fix incorrect font base size
This issue is automatically created based on existing pull request: magento/magento2#30363: Fix incorrect font base size 

---------

### Description (*)

This PR aim to correct font base magento frontend back to 16px
Change @font-size-ratio__base
Before: ratio 1.4
After: ratio 1.6
This is default font size browser
1em = 16px = 1rem

some element such as input, browser will zoom if the font-size is less than 16px.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by mrtuvn**
### Description (*)

This PR aim to correct font base magento frontend back to 16px
Change @font-size-ratio__base
Before: ratio 1.4
After: ratio 1.6
This is default font size browser
1em = 16px = 1rem

some element such as input, when user interactive with components browser will zoom if the font-size is less than 16px.

Also updated button line height . Add new variable @button__line-height__base with unitless value

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#30367: Fix incorrect font base size

### CODE REVIEW

**Comment by mrtuvn** [comment]
@krzksz @larsroettig what do you guys think about this PR

**Comment by sidolov** [comment]
@magento create issue

File: lib/web/css/source/lib/variables/_buttons.less [hunk 1]
```diff
@@ -103,3 +104,4 @@
 @button__font-size__s: 11px;
 @button__line-height__s: @button__font-size__s + 1;
 @button__padding__s: @indent__xs 8px;
+
```
Comment (mrtuvn) [inline lib/web/css/source/lib/variables/_buttons.less]: we should use unitless value for line-height when possible. In this we set default line height for button init value one

File: lib/web/css/source/lib/variables/_typography.less [hunk 2]
```diff
@@ -26,17 +26,17 @@
...
 //  Sizes
 @root__font-size: 62.5%; // Defines ratio between root font size and base font size, 1rem = 10px
-@font-size-ratio__base: 1.4; // Defines ratio of the root font-size to the base font-size
+@font-size-ratio__base: 1.6; // Defines ratio of the root font-size to the base font-size
 
 @font-size-unit: rem; // The unit to which most typography values will be converted by default
 @font-size-unit-ratio: unit(@root__font-size * 16/100); // Ratio of the root font-size to the font-size unit
 @font-size-unit-convert: true; // Controls whether font-size values are converted to the specified font-size unit
 
 @font-size__base: unit(@font-size-unit-ratio * @font-size-ratio__base, px); // Base font size value in <b>px</b>
```
Comment (mrtuvn) [inline lib/web/css/source/lib/variables/_typography.less:35]: @font-size__base now have value 16px

File: lib/web/css/source/lib/variables/_buttons.less (hunk 1 above)
Comment (mrtuvn) [reply lib/web/css/source/lib/variables/_buttons.less]: @Leland what do you think about this

File: lib/web/css/source/lib/_buttons.less [hunk 3]
```diff
@@ -73,7 +73,7 @@
     .lib-css(padding, @_button-padding);
     .lib-css(width, @_button-width);
     .lib-font-size(@_button-font-size);
-    .lib-line-height(@_button-line-height);
+    .lib-css(line-height, @_button-line-height);
```
Comment (Leland) [inline lib/web/css/source/lib/_buttons.less]: This is just working around #25592 – if there are downstream themes that use `.lib-line-height(@_button-line-height)` directly (probable) they'll be very surprised when things look bad on updating.

I personally don't think that `@button__line-height__base` should be updated to unitless until that issue is resolved.

Comment (mrtuvn) [reply lib/web/css/source/lib/_buttons.less]: i have reverted back this file . We should do this in another seperate PR

**Comment by mrtuvn** [comment]
Closed due this can PR introduced new visual. Font changed make text screen bigger than previous

**Pull request closed without merging**